cmd/viam-agent/main_windows.go

```diff
@@ -41,7 +46,7 @@ func main() {
 	if inService, err := svc.IsWindowsService(); err != nil {
 		panic(err)
 	} else if !inService {
-		println("no service detected -- running as normal process")
+		globalLogger.Info("no service detected -- running as normal process")
```

Comment:
Check with @abe-winter on this. "println()" prints to STDERR I believe, not sure if changing this into a log line will prevent it from going into the windows service logs properly or something.